colorclass.py

```diff
@@ -478,6 +488,14 @@ def zfill(self, width):
         return Color().join(split)
 
 
+# Add static convenience methods for colors -- e.g.:
+# - Color.green(s, auto=True)
+# - Color.red(s, auto=True)
+# etc.
+for color in colors:
+    setattr(Color, color, functools.partial(Color.colorize, color))
```

Comment:
Can you explicitly define multiple classmethods instead of defining them dynamically upon import? This might break pycharm inspections.